LICENSE

```diff
@@ -1,6 +1,7 @@
 SPDX-License-Identifier: MIT
 
 Copyright (C) 2024 Savoir-faire Linux Inc. (<www.savoirfairelinux.com>).
+Copyright (C) 2024 iris-GmbH infrared & intelligent sensors
```

Comment:
About that, seem legit to me but I need approval from the original maintainer of the project at Savoir-Faire Linux. He come back next week, I will keep you updated near friday 19th

Reply:
No worries, we will figure it out :+1: